dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -636,6 +639,7 @@ def to_csv_row(user)
           school_city,
           school_state,
           school_zip_code,
+          created_at.strftime('%B %e'),
```

Comment:
Are you sure we want this format (Month day)? That is not sortable. I'd suggest we use the [ISO8601](https://xkcd.com/1179/) date like we do here: https://github.com/code-dot-org/code-dot-org/blob/900ee936a46b491b9dd571ca0c42c4fc049da068/dashboard/app/models/pd/application/application_base.rb#L304-L307

Reply:
Good question. This is how we display it on list view, which is why I did it this way, but sorting is probably more important.

Reply:
And there, it's still sortable since the formatting is only a [display transform](https://github.com/code-dot-org/code-dot-org/blob/900ee936a46b491b9dd571ca0c42c4fc049da068/apps/src/code-studio/pd/application_dashboard/quick_view_table.jsx#L92)